composer.json

```diff
@@ -42,8 +42,7 @@
     "phpstan/phpstan-deprecation-rules": "^1.1",
     "phpstan/extension-installer": "^1.3",
     "szepeviktor/phpstan-wordpress": "*",
-    "automattic/jetpack-changelogger": "^4.2",
-    "slope-it/clock-mock": "^0.4.0"
```

Comment:
yep clock mock has been removed - i guess accidentally

Reply:
That was unintentional - I'll put it back